client/landing/subscriptions/components/add-sites-form/add-sites-form.tsx

```diff
@@ -14,13 +15,20 @@ type AddSitesFormProps = {
 	onAddFinished: () => void;
 };
 
+type SubscriptionError = {
+	error?: string;
+	message?: string;
+};
+
 const AddSitesForm = ( { onAddFinished }: AddSitesFormProps ) => {
 	const translate = useTranslate();
 	const [ inputValue, setInputValue ] = useState( '' );
 	const [ inputFieldError, setInputFieldError ] = useState< string | null >( null );
 	const [ isValidInput, setIsValidInput ] = useState( false );
+	const [ showLoginDialog, setShowLoginDialog ] = useState( false );
 	const { showErrorNotice, showWarningNotice, showSuccessNotice } = useAddSitesModalNotices();
 	const recordSiteSubscribed = useRecordSiteSubscribed();
+	const { isLoggedIn } = SubscriptionManager.useIsLoggedIn();
```

Comment:
What is the difference between using `SubscriptionManager` for this, as opposed to `isUserLoggedIn` selector from calypso state?

Reply:
I'm 90% certain they're functionally identical in that `SubscriptionManager.useIsLoggedIn();` also checks the state to determine if the user is logged in; it's just conveniently accessible since we're already importing SubscriptionManager.